qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -65,31 +60,22 @@
         "PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals", "PMD.GodClass"
     }
 )
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Not clear for me, why should we restrict execution to per_class? If there are any potential problems they can appear for another classes also.

Reply:
Now there is no need for this, I have removed the annotation